ClientFiles/Resources/GameCollectionConfig.ini

```diff
@@ -1,19 +1,4 @@
 ; List of custom CnCNet games / mods.
 [CustomGames]
-;0=CustomGame
+0=CustomGame
 
-[CustomGame]
-; Internal game name, should match LocalGame in ClientDefinitions.ini (case-insensitive). Maximum length is 4 characters.
-InternalName=cg
-; Game's displayed name.
-UIName=Custom Game
-; Lobby / Chat IRC channel. Must be a valid IRC channel name. For an example: #cncnet-cg.
-ChatChannel=
-; Hosted games broadcast IRC channel. Must be a valid IRC channel name. For an example: #cncnet-cg-games.
-GameBroadcastChannel=
-; Name of the client launcher executable - currently unused.
-ClientExecutableName=YRLauncher.exe
-; Registry key - currently unused.
-RegistryInstallPath=HKLM\Software\CustomGame
-; Filename of game's icon in Resources folder.
-IconFilename=customgameicon.png

```

Comment:
What is current D2K channel in CnCNet's IRC?

Reply:
no, don't need that, the client already has Dune config hardcoded

Reply:
> no, don't need that, the client already has Dune config hardcoded

You mean `LocalGame=d2k` in `ClientDefinitions.ini` is enough to enable the D2K channel for the client?